tests/cases/06_gridfs.ts

```diff
@@ -113,3 +114,21 @@ testWithClient(
     assert(!file[0]);
   },
 );
+
+// https://www.mongodb.com/docs/manual/reference/command/createIndexes/#considerations
+testWithClient(
+  "GridFS: Creating indexes - skip index creation on same index keys",
+  async (client) => {
+    const addAsset = async (index: number) => {
+      const bucket = new GridFSBucket(client.database("test"), {
+        bucketName: "sameKeys",
+      });
+      const upstream = await bucket.openUploadStream(`test-asset-${index}`);
+      const writer = upstream.getWriter();
+      await writer.write(new TextEncoder().encode(`[asset${index}]`));
+      await writer.close();
+    };
+    await addAsset(0);
+    await addAsset(1);
```

Comment:
Where is the assert in this code? Shouldn’t the test case somehow check if the indexes where skipped?